rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/CodeFormatterService.java

```diff
@@ -0,0 +1,10 @@
+package com.regnosys.rosetta.formatting2;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.eclipse.emf.ecore.resource.Resource;
+
+public interface CodeFormatterService {
```

Comment:
This name is perhaps a bit generic - it doesn't really tell you what it does. What about `ResourceFormatterService`?